GiniSwitchSDK/GiniSwitchSDK/Classes/GiniSwitchSdk.swift

```diff
@@ -40,23 +46,37 @@ public class GiniSwitchSdk {
         return configuration.appearance
     }
     
+    /*
+     * Creates an empty GiniSwitchSdk instance. Mostly shouldn't be used. Use
+     * `init(clientId: String, clientSecret: String, domain: String)` instead
+     */
     public init() {
```

Comment:
Actually it does. I think I didn't use it because then they will be able to add one parameter, but not the other. But I guess, that wouldn't be so important. I'll merge them into one initialiser.